File: portal/portal/App_Code/LogOperacao.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace portal.App_Code
{
    public class LogOperacao
    {
        private double _codigo;
        private string _tabela;
        private double _codigoChave;
        private double _codigoUsuario;
        private string _data;
        private TiposOperacaoes _tipoOperacao;

        public double Codigo
        {
            get
            {
                return _codigo;
            }

            set
            {
                _codigo = value;
            }
        }

        public string Tabela
        {
            get
            {
                return _tabela;
            }

            set
            {
                _tabela = value;
            }
        }

        public double CodigoChave
        {
            get
            {
                return _codigoChave;
            }

            set
            {
                _codigoChave = value;
            }
        }

        public double CodigoUsuario
        {
            get
            {
                return _codigoUsuario;
            }

            set
            {
                _codigoUsuario = value;
            }
        }

        public string Data
        {
            get
            {
                return _data;
            }

            set
            {
                _data = value;
            }
        }

        public TiposOperacaoes TipoOperacao
        {
            get
            {
                return _tipoOperacao;
            }

            set
            {
                _tipoOperacao = value;
            }
        }

        public enum TiposOperacaoes {Inserir, Alterar, Excluir};

    }
}

File: portal/portal/App_Code/LogOperacao.php
<?php
###################################################################################################################
//LOG OPERAÇÃO: Guarda LOGS do sistema (Ações feitas por usuários).
####################################################################################################################

namespace portal\App_Code {
    class LogOperacao {
//ATRIBUTOS----------------------------------------------------------------------------------------------------------		
        private $codigo;
        private $tabela;//String
        private $codigoChave;
        private $codigoUsuario;
        private $data;//String
        private $tipoOperacao;//(1-Inserir, 2-Excluir, 3-Alterar) 
//CONSTRUTOR----------------------------------------------------------------------------------------------------------			
		public function __construct (){}
        
        public function setLogOperacao($codigo,$tabela,$codigoChave,$codigoUsuario,$data,$tipoOperacao){
			$this -> codigo = $codigo;
			$this -> tabela = $tabela;
			$this -> codigoChave = $codigoChave;
			$this -> codigoUsuario = $codigoUsuario;
			$this -> data = $data;
			$this -> tipoOperacao = $tipoOperacao;			
		}
//SETS-----------------------------------------------------------------------------------------------------------------
		Public function setCodigo ($codigo){
			$this -> codigo = $codigo;
		}
		Public function setTabela($tabela){
			$this -> tabela = $tabela;
		}
		Public function setCodigoChave($codigoChave){
			$this -> codigoChave = $codigoChave;
		}
		Public function setCodigoUsuario($codigoUsuario){
			$this -> codigoUsuario= $codigoUsuario;
		}
		Public function setData ($data){
			$this -> data= $data;
		}	
		Public function setTipoOperacao($tipoOperacao){
			$this -> tipoOperacao = $tipoOperacao;
		}		
//GETS-----------------------------------------------------------------------------------------------------------------			
    	Public function getCodigo(){
			return $this -> codigo;
		}
		Public function getTabela(){
			return $this -> tabela;
		}
		Public function getCodigoChave(){
			return $this -> codigoChave;
		}
		Public function getCodigoUsuario(){
			return $this -> codigoUsuario;
		}
		Public function getData(){
			return $this -> data;
		}	
		Public function getTipoOperacao(){
			return $this -> tipoOperacao;
		}
	}
}
?>